examples/asr/conf/conformer/conformer_ctc_bpe.yaml

```diff
@@ -188,7 +184,7 @@ trainer:
   strategy: ddp
   accumulate_grad_batches: 1
```

Comment:
perhaps mention that grad accum could be used to fix global batch size (e.g. 2k recommended when training from scratch) - so that the runs could be conducted on any number of nodes

Reply:
I added these recommendations on top.